index.js

```diff
@@ -10,6 +10,7 @@ var formatter = require('stylelint').formatters.string;
 var runCompilation = require('./lib/run-compilation');
 var LintDirtyModulesPlugin = require('./lib/lint-dirty-modules-plugin');
 var defaultFilesGlob = require('./lib/constants').defaultFilesGlob;
+var webpackFormatter = require('./lib/formatter-webpack');
```

Comment:
minor nit, the filename should be kebab-cased version of the export, i.e. `webpack-formatter`.

Reply:
if we put the context after the subject, then `lint-dirty-modules-plugin` should be named `dirty-modules-plugin-lint` as well ;) that seems inconsistent with the rest of the codebase